### PR TITLE
[IMP] inventory: update 3 step receipts doc

### PR DESCRIPTION
**Pull request by brse-odoo**
This PR is an [IMP] to the `receipts_three_steps` doc in Odoo's Inventory documentation, specific to 17.0.

Here's what I did:
- Retooled some of the instructions/flow for 17.2 update that affected all multi-step routes
- Replaced all screenshots for 17.2 UI, removed some screenshots no longer relevant

**Note:** _This PR was only to update small instructions and replace screenshots. A second, targeted PR for a more robust rewrite will follow once this is merged._

[Project Task Link](https://www.odoo.com/web#id=3877918&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst
Comment (hojo-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst:65]: You could also use an abbreviation tag for "purchase order" if this isn't the first time the term has been mentioned in the doc.
```suggestion
Once the purchase order (PO) is confirmed, a receipt (`WH/IN`) operation is generated and ready to
```

**Review by hojo-odoo — APPROVED** [review]
Hey @brse-odoo,

Nice job on this update. I just had one small suggestion.

Thanks,
hojo

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst:65]: Suggestion implemented ✅ - I'll definitely add in a replace abbr tag for all the POs throughout the doc in a full rewrite PR to follow!

**Comment by brse-odoo** [comment]
Hi @odoo/us-doc-review, this PR is ready for final content review. Thank you!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst:58]: Consider reverting here, this revision is a bit too zoomed in so we're missing navigational context from the surrounding UI elements.

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst:82]: Sanity check: These states, **Ready** and **Waiting Another Operation** are no longer relevant to 3-step receipts, correct?

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst:94]: All three of these paragraphs begin with the word **Click**.
Consider using more varied or transitional language, please.

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst:77]: This screenshot is a bit too zoomed in and removes the context of the surrounding UI elements.

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst:111]: Same here: **Click**.

**Review by StraubCreative — APPROVED** [review]
Hi @brse-odoo 
Approving with comments.

Please address the zoomed-in screenshots:
- receipts-three-steps-internal-transfer.png
- receipts-three-steps-receipt-form.png
- receipts-three-steps-second-transfer.png
- receipts-three-steps-smart-button.png

...as well as the few sections that repetitively begin with **Click**.

Then can pass to SALI, thank you!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst:82]: Yes, that's correct. 3-step receipts and deliveries now generate the operation only when it's ready, instead of listing all 3 at once like before.

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst:94]: Changed some words in the second and third paragraphs per the instructions from this suggestion 🙂 Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst:111]: Changed some words in the second and third paragraphs per the instructions from this suggestion 🙂 Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst:58]: Screenshot replaced with new zoomed-out one that shows navigational UI and highlights key areas. Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst:77]: Screenshot replaced with new zoomed-out one that shows navigational UI and highlights key areas. Suggestion implemented ✅

**Comment by brse-odoo** [comment]
> Hi @brse-odoo Approving with comments.
> 
> Please address the zoomed-in screenshots:
> 
> * receipts-three-steps-internal-transfer.png
> * receipts-three-steps-receipt-form.png
> * receipts-three-steps-second-transfer.png
> * receipts-three-steps-smart-button.png
> 
> ...as well as the few sections that repetitively begin with **Click**.
> 
> Then can pass to SALI, thank you!

Hi @samueljlieber this PR has passed us-doc-review and is ready for your lookover before merge, thank you!

**Review by samueljlieber — APPROVED** [review]
Hi @brse-odoo, nice PR! Your changes look good to me 👍
..
@robodoo r+

**Pull request closed without merging**
